### ISSUE TITLE
JSX tag automatically re-closes after deletion

### ISSUE THREAD

**Issue by dceddia** [asvetliakov/vscode-neovim#212]
I had a problem where a JSX tag was being auto-closed as soon as I deleted the closing tag. It looks like this:

![auto-closing-tag](https://user-images.githubusercontent.com/505704/75820638-ba325a00-5d6a-11ea-9500-02f6668a1a87.gif)

My first attempt at fixing it was to turn off VSCode's "Auto Closing Tags" features - in the version I'm running (1.42.1) there are auto-closing features for HTML, JavaScript, and TypeScript. I disabled them all and it was still happening.

Then I went looking through my extensions and found [Auto Close Tag](https://marketplace.visualstudio.com/items?itemName=formulahendry.auto-close-tag) (which I guess I installed before VSCode supported auto-closing tags in JSX? I'm not sure.)

Long story short, disabling that extension fixed the problem. After that I re-enabled VSCode's own auto-closing tag feature for HTML, JS, and TS, and those work correctly!

I'm writing this up here just in case anyone else stumbles into the same problem. Check your extensions!

(Side note, thanks for your work on vscode-neovim! It feels like I'm editing at the speed of vim again. So awesome :)
